steps/executexcuitestonbrowserstack/0.0.1/step.yml

```diff
@@ -0,0 +1,79 @@
+title: executeXCUITestOnBrowserstack
```

Comment:
The users will be able to search after the step by its title. The current title is not the best in the manner of UX. Please change it to something like: `Browserstack iOS` or `Browserstack iOS UI Test`.

Also the step ID is a little too long and makes a bit hard for the users to define it in their bitrise.yml. Currently they have to use the `executexcuitestonbrowserstack` as the step ID f they want to use it. Please choose a shorter, more descriptive step Id and title.